en/lessons/basics/testing.md

```diff
@@ -38,7 +38,7 @@ Finished in 0.03 seconds
 ```
 
 Why there are two tests in output? Let's look at `lib/example.ex`.
-Mix created there another test for us, some doctest.
+Mix has automatically created a doctest.
```

Comment:
This one conflicts with https://github.com/elixirschool/elixirschool/pull/1973. Just a note to the reviewer.